app/models/ability.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    can :read, :all
+    can :edit, :all
+    can :index, :all
+    can :destroy, :all
```

Comment:
@basti  I have problems with showing error message in case that action is not allowed, because CanCan rescue message did not work as I wanted, so I do not let user from front-end to do unpermitted actions because i remove all button actions (show, create, edit and delete) for user that is not permitted, is that ok?